### DISCUSSION TITLE
Update for Buds FE?

### DISCUSSION THREAD

**Discussion by Kazekage-Gaara** [timschneeb/GalaxyBudsClient#440] [General]
have an update planned to be able to use the app with Galaxy Buds FE?

**Comment by passivesnowflake** [comment]
Still waiting on that myself! Also, firmware update. Hope it gets updated in next version

**Comment by FogShard** [comment]
Any word on a galaxy buds fe update?

**Comment by muelsyse-az** [comment]
Galaxy Buds FE support has been added a while back and it works perfectly with mine. There are firmware updates available if needed but I haven't checked if there have been newer firmware since the last one.
